pkg/sliexporter/sli_reconciler/reconciler.go

```diff
@@ -31,7 +31,8 @@ type Reconciler struct {
 	nn                 types.NamespacedName
 	startupGracePeriod time.Duration
 	fetchProberFor     func(context.Context, Service) (probes.Prober, error)
-	client             client.Client
+	// TODO: multiple clients
```

Comment:
TODO?

Reply:
We can crape that, I ended up not using multiple clients at all.